meetings/2022-05-09.md

```diff
@@ -0,0 +1,61 @@
+# W3C Solid Community Group: Test Suite Panel
+
+* Date: 2022-05-09T14:00:00Z
+* Call: https://meet.jit.si/solid-test-suite
+* Chat: https://gitter.im/solid/test-suite
+* Repository: https://github.com/test-suite-panel
+
+
+## Present
+* [Sarven Capadisli](https://csarven.ca/#i)
+
+---
+
+## Announcements
+
+### Meeting Recordings and Transcripts
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Join queue to talk.
+
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* name
+
+
+### Introductions
+* name: text
+
+---
+
+## Topics
+
+* SC: The Test Suite Panel is a "horizontal" panel. We need to revisit/clarify our understanding of the Test Suite Panel alongside other panels in the CG as well as contributors to specs and implementers.
+
+
+### Test Suite Panel Charter
+* SC: We need to stabilise a mutual understanding and a coordination plan. To advance our work, we need to agree on a charter that details its goals, scope, deliverables, decision policy (e.g., along the lines of https://github.com/solid/process/blob/main/notifications-panel-charter.md ).
+* SC: Here are some proposed actions. In these proposals, it is assumed that:
```

Comment:
While I value your work proposing this charter, you can't phrase your own proposals as assumptions ahead of the meeting, so please change "it is assumed that" to "I assume that", to make it clear that this text is coming from you.

Reply:
It is *I* that is making the proposal and so it is *I* that is clarifying the assumptions. I thought that was evident but I've updated the agenda to say "I" in any case.